src/FieldDefinition.hack

```diff
@@ -1,5 +1,13 @@
 namespace Slack\GraphQL;
 
+type ArgumentDefinition = shape(
+    'name' => string,
+    'type' => Types\IInputType,
+    ?'description' => string,
+    ?'default_value' => mixed,
+    ?'deprecation_reason' => string,
+);
```

Comment:
Depending on whether we end up reusing any of this for introspection, we may not need most of these here (name is redundant because we have the names as dict keys, only type and `'has_default' => bool` would be needed for validation)

Reply:
Good point, I'll remove most of this in a subsequent PR.